demo/package-lock.json

```diff
@@ -0,0 +1,9867 @@
+{
```

Comment:
I don't think we need the `package-lock.json` file versioned for now.

Reply:
Oh this has already been removed